indico/modules/events/api.py

```diff
@@ -614,6 +616,10 @@ def _build_event_api_data(self, event):
             data['occurrences'] = fossilize(self._calculate_occurrences(event, self._fromDT, self._toDT,
                                             pytz.timezone(config.DEFAULT_TIMEZONE)),
                                             {Period: IPeriodFossil}, tz=self._tz, naiveTZ=config.DEFAULT_TIMEZONE)
+        # check whether the plugins want to add/override any data
+        for update in values_from_signal(
+                signals.event.metadata_postprocess.send('http-api', event=event, data=data), as_list=True):
+            data.update(update)
```

Comment:
what about simply letting the signal mutate data instead of using a return value? nothing stops someone from doing this right now anyway..

Reply:
True.

Reply:
At the same time, this kind of makes things a little bit more explicit, doesn't it.